azurelinuxagent/ga/exthandlers.py

```diff
@@ -304,10 +319,52 @@ def cleanup_outdated_handlers(self):
             if os.path.isfile(pkg):
                 try:
                     os.remove(pkg)
-                    logger.verbose("Removed extension package {0}".format(pkg))
-                except OSError as e:
-                    logger.warn("Failed to remove extension package {0}: {1}".format(pkg, e.strerror))
-   
+                    logger.verbose("Removed extension package "
+                                "{0}".format(pkg))
+                except Exception as e:
```

Comment:
You broadened the exception that is handled, and I don't know why.  This is a breaking change, why?

Reply:
Sorry, no idea why this was updated.  I don't know if I accidently did it or Pawel.  Reverting as there's no reason for a modification there.